added use cases and resources in about.md for unconditional image generation task (#464)
huggingface/hub-docs: tasks/src/unconditional-image-generation/about.md

## Use cases

> In the cartoon anime industry, it is normally time-consuming to draw images of characters, objects 

Reviewer: ```suggestion
Drawing images of characters, objects, and landscapes is time-consuming in creating animations
```

> as well as landscapes because there is a huge demand on images and every little detail of each image

Reviewer: ```suggestion
because drawing details of each image is demanding.
```

needs to be taken into account. Furthermore, hiring high qualified artists cannot always be possible. 
In this case, unconditional image generation models can be used to generate many anime images economically.

> Choosing a hairstyle or makeup that fit one's face is not always easy without trying them before. 

Reviewer: Are you sure? As said, unconditional image generation is usually used to generate image from simple noise (and the noise is generated from a seed) you don't really input anything like you'd do with text-to-image models.

Author: thank you for your comments i will do more research and let you know the questions i may have

These models can help with this by generating images with different hairstyle or makeup from input ones in advance.

One can also gain higher quality of input images with the help of these models.

> ## Task variants

Reviewer: Can you copy and paste this template and make changes to the sections that you wish to change? https://github.com/huggingface/hub-docs/blob/main/tasks/src/placeholder/about.md
